mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/TiltakstypeRepository.kt

```diff
@@ -97,6 +94,12 @@ class TiltakstypeRepository(private val db: Database) {
                 Status.AKTIV -> "" to "(now()::timestamp >= fra_dato and now()::timestamp <= til_dato)"
                 Status.PLANLAGT -> "" to "(now()::timestamp < fra_dato)"
                 Status.UTFASET -> "" to "(now()::timestamp > til_dato)"
+            },
+            tiltakstypeFilter.kategori to tiltakstypeFilter.kategori?.let {
+                when (it) {
+                    Tiltakstypekategori.GRUPPE -> "tiltakskode in ${Gruppetiltak.somSqlListe()}"
```

Comment:
Hm, dette blir en potensiell inngang til sql-inections (uten at det er noen fare enda).

Nå har vi kontroll på  `Gruppetiltak.somSqlListe()` i koden slik at vi kan sørge for at dette ikke skjer, men vi kan også skrive spørringen som `Tiltakstypekategori.GRUPPE -> "tiltakskode in :gruppetiltak"` og legge til `"gruppetiltak" to Gruppetiltak.somSqlListe()` i `parameters` for å oppnå det samme, men samtidig skiller verdier fra selve sql-stringen.

Reply:
Jeg mener jeg prøvde det i går, og fikk det ikke helt til. Jeg kan prøve på nytt nå og se om jeg blir noe klokere.

Reply:
Hmm, det var jo litt kjipt da. Antagelig blir hele resultatet tolket som en string ja, og da funker det sikkert dårlig :disappointed: